python3.8/cookiecutter-aws-sam-hello-python/{{cookiecutter.project_name}}/tests/integration/test_api_gateway.py

```diff
@@ -0,0 +1,55 @@
+import os
+from unittest import TestCase
+
+import boto3
+import requests
+
+"""
+Make sure env variable AWS_SAM_STACK_NAME exists with the name of the stack we are going to test. 
+"""
+
+
+class TestApiGateway(TestCase):
+    api_endpoint: str
+
+    @classmethod
+    def get_stack_name(cls) -> str:
+        stack_name = os.environ.get("AWS_SAM_STACK_NAME")
```

Comment:
Just curious, do we use this name elsewhere too?

Reply:
this is new, and original